dz/login.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"/>
+	<meta http-equiv="Content-type" content="text/html; charset=utf-8"/>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+    <title>Login</title>
+    <link rel="stylesheet" type="text/css" media="screen" href="css/loginStyle.css"/>
+</head>
+<body>
+    <div id="container">
+		<form>
+            <h2>Логин</h2>
+            <div class="button-social">
+                <button id="vkontakteButton"><img src="images/vk.svg">Войти через Вконтакте</button>
+                <button id="facebookButton"><img src="images/fb.svg">Войти через Facebook</button>
+            </div>
+            <div id="lineDiv">
+                <span id="lineSpan">
+                    или
+                </span>
+            </div>
+            <div class="data-input">
+                <div class="input-area">
+                    <input name="Email" type="name" placeholder="Email">
+                    <input name="password" placeholder="Пароль">
+                    <span class="alert">Введите пароль</span>
+                </div>
+                    <button id="submit"><img src="images/mailru.svg">Войти через почту</button>

```

Comment:
Тег `img` надо не забыть закрыть. Он самозакрывающийся, так что можно так:
```suggestion
                    <button id="submit"><img src="images/mailru.svg" />Войти через почту</button>
```